js/templates/template-page-weather.js

```diff
@@ -30,7 +30,7 @@ function initHTMLpageWeather () {
 			</div>
 			<div class="pagedetails__subheader">
 				<h4>Custom type</h4>
-				<a class="tipsy-w showtip pictos" original-title="Input URL to your PNG when &quot;Custom&quot; is selected above">?</a>
+				<a class="tipsy-w showtip pictos" original-title="Input URL to your PNG when &quot;Custom&quot; is selected above. Broken URLs will appear as rain.">?</a>
```

Comment:
hm, this is a bit subtle, and sticking the "solution" in the tooltip isn't great. Maybe we grab the classic [missing texture](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSfGGlO7vyn7hh8pwkPxJe4JJjaLIFiZa6BYxBDmozG-eNp5z6r3ILcV_6A2ZwXeGcCY7A) image and chuck it in as the fallback?